debian/patches/update-unit.patch

```diff
@@ -0,0 +1,23 @@
+Update ferm.service from archlinux
+
+https://projects.archlinux.org/svntogit/community.git/plain/trunk/ferm.service?h=packages/ferm
+
+Except, we still use sbin instead of bin.
+
+--- a/ferm.service
++++ b/ferm.service
+@@ -1,10 +1,14 @@
+ [Unit]
+ Description=for Easy Rule Making
++After=ipset.service
```

Comment:
Is it technically necessary to start ipset before ferm? If it works without then I suggest dropping this.